packages/cli/src/lib/commands/space.ts

```diff
@@ -148,17 +148,33 @@ export const space: CommandDefinition = (program) => {
 
     accessKeyCmd.command("revoke")
         .description("Revokes Access Key in active Space")
-        .argument("<access_key>", "Access Key id")
-        .action(async (accessKey: string) => {
+        .option("--id <id>", "Hub Id")
```

Comment:
missing id argument validation- now it can be any invalid string.

Check topics command how to use argParser (same works for optional arg) i.e.:
`const topicNameArgument = new Argument("<topic-name>").argParser(validateTopicName);`